content/projects/gsoc/gsoc2018-project-ideas.adoc

```diff
@@ -93,17 +93,21 @@ link:https://github.com/oleg-nenashev[Oleg Nenashev],
 link:https://github.com/christ66[Steven Christou]
 
 
-== Integration plugin(s) for Electronic Design Automation tools
+== Plugin(s) for Electronic Design Automation tools
 
 The idea is to create a Jenkins plugin for one of widely used EDA tools.
 Both ASIC or FPGA design flow are acceptable, the tool should be proposed by the potential student.
 Open-source EDA tools would be preferable (e.g. Yosys, FuseSoC, ArachnePnR, icetools), but we also consider
 conditionally-free tools (like FPGA design EDAs).
 
-For example such plugin could report FPGA resource utilization per build as well as the project trend.
-Other example: timing report trend.
 
-Integrating UVM reports into Jenkins build and project pages is also an example of such plugin.
+Examples of tool integration:
+
+* Tool launch and publishing steps for Free-style and/or Pipeline jobs
+* Integration with Warnings Plugin for report parsing.
+* Reporting of FPGA resource utilization (per build + trends)
+* Timing report trend publishing
+* Integrating UVM reports into Jenkins build and project pages
```

Comment:
@oleg-nenashev As a general comment which is not important at this early phase, I would suggest making multiple build steps so users could use the reporting steps without being forced to use the launching steps.

Reply:
Yup. I think it should be clarified on the project proposal / design stage